Before-After-Python/After/if.spy

```diff
@@ -0,0 +1,15 @@
+x: int = 10
+
+if x > 10:
+    print("more")
+elif x < 10:
+    print("less")
+else:
+    print("equal")
+
+fact: bool = False
+
+if bool:
```

Comment:
if boolean is false, then print "True"?

Reply:
i made a mistake. I wrote 'bool' but should 'fact'.
Answer to your question: No. first the body of 'if' will not be printed, the body of 'else' will.

Reply:
I forgot to point about variable too